npm-shrinkwrap.json

```diff
@@ -6294,8 +6294,7 @@
         },
         "ansi-regex": {
           "version": "2.1.1",
-          "bundled": true,
-          "optional": true
```

Comment:
Do you know why `optional` is being removed from all of these? Was this accidentally committed?

Reply:
Not really. I am not sure at what point the file got accidentally commited, but since I am not using any new packages, I'm avoiding to commit it.